tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -465,7 +465,7 @@ def eltwise_relu_max(
 @setup_host_and_device
 def std_hw(x, *args, device, dtype, layout, input_mem_config, output_mem_config, **kwargs):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttl.tensor.std_hw(t0, output_mem_config=output_mem_config)
+    t1 = ttnn.std_hw(t0)
```

Comment:
Yet another example of a function that deviates from PyTorch.  Not part of this PR obviously but we need to get these fixed soon.

Reply:
Yes this will be handled when we work on M6.